.eslintignore

```diff
@@ -0,0 +1,10 @@
+.eslintrc.js
```

Comment:
Can be ignored since it's auto generated

Reply:
added to .gitignoore